### DISCUSSION TITLE
swanctl.conf seems not doing close_action

### DISCUSSION THREAD

**Discussion by chesskuo** [strongswan/strongswan#948] [General]
**System (please complete the following information):**
 - OS: `Debian 11`
 - Kernel version (if applicable): `5.10.0-11-amd64`
 - strongSwan version(s): `5.9.1`
 - Tested/confirmed with the latest version: `no`

**Describe the bug**
When IPsec ESP timeout, the tunnel will close and not reconnect automatically.
This connection was still in ESTABLISHED state but not go to INSTALLED (means the tunnel not up).

**To Reproduce**
I set the esp timeout very short (as 30s) to try it and use this config on each sides.

```
connections {
    peer_bbb {
        proposals = aes128-sha1-modp2048
        version = 1
        local_addrs = aaa
        remote_addrs = bbb
        dpd_timeout = 120
        dpd_delay = 30
        rekey_time = 86400s
        local {
            auth = psk
        }
        remote {
            id = "bbb"
            auth = psk
        }
        children {
            peer_bbb_tunnel_0 {
                esp_proposals = aes128-sha1-modp1024
                life_time = 30s
                mode = transport
                start_action = start
                dpd_action = start
                close_action = start
            }
        }
    }
}

secrets {
    ikey_bbb {
        id-local = aaa
        id-remote = bbb
        secret = "myipseckey"
    }
}

```

**Expected behavior**
The tunnel will reconnect automatically.

**Logs/Backtraces**

**Additional context**
If I use ipsec.conf and set the `closeaction=restart` it works, but I migrated it to swanctl.conf, it was broken.

**Comment by tobiasbrunner** [comment]
`close_action` has nothing to do with `life_time`. And setting `life_time` without also changing `rekey_time` is a bad idea. Rather set just the latter (`life_time` will then be based on it) to configure rekeying (see [this](https://docs.strongswan.org/strongswan-docs/5.9/config/rekeying.html) for details).
